jyulb/fileio/isothermal/jyu_engine.py

```diff
@@ -49,10 +52,11 @@ def __init__(self):
         """Initialize and set default parameters for CM, BC, SP, and SD."""
         # Definition of CM, SP, BC, and SD data components
         self._data = {}
-        self._lattice = None
+        self._proxy_lattice = None
         self.CM = {}
         self.SP = {}
-        self.BC = DataContainer()
+        self.SD = {}
```

Comment:
You might be right: modifying SD should not be available to the user in other ways than with the provided methods.